lib/registry/client.go

```diff
@@ -124,6 +124,14 @@ func (c DockerRegistryClient) Pull(tag string) (*image.DistributionManifest, err
 			}
 		})
 	}
+	l := manifest.GetConfigDigest()
+	workers.Do(func() {
+		if _, err := c.PullLayer(l); err != nil {
+			multiError.Add(fmt.Errorf("pull config %s: %s", l, err))
```

Comment:
this message only shows when pull/push fails thou. Are you sure this is what you want..